_includes/macros/mobile-carousel.html

```diff
@@ -0,0 +1,50 @@
+
+{# ==========================================================================
+
+   mobile_carousel.render()
+
+   ==========================================================================
+
+   Description:
+
+   Render HTML for a carousel of slides with an image, title, and description.
+   Argument is:
+
+   slides: An array of data to populate the carousel slides with.
+           The structure looks like:
+           {
+           'image_sizes_class': <A image sizing class for responsive sizes>,
+           'image_base_url':    <Base URL path for images>,
+           'data': [{
+              'image_filename': <Carousel slide image filename>,
+              'title':          <Carousel slide filename>,
+              'desc':           <Carousel slide description>
+              }, {
+              'image_filename': <Carousel slide image filename>,
+              'title':          <Carousel slide filename>,
+              'desc':           <Carousel slide description>
+              }]
+           }
+
+   ========================================================================== #}
+
+{% macro render( slides ) %}
```

Comment:
My preference is for spaces, since that's the only standard we've had (in JS). But since it's not the way CSS or Jinja files currently are, go without for the time being.